src/components/Explore/LocationSelector.tsx

```diff
@@ -36,81 +26,65 @@ const LocationSelector: React.FC<{
     onChangeSelectedRegions(newRegions);
   };
 
-  const onClickButton = () => setModalOpen(!modalOpen);
+  const onClickButton = () => {
+    if (!modalOpen) {
+      trackEvent(
```

Comment:
adds tracking for opening the modal. we'll keep the tracking for closing the modal too, since [the tracking props](https://github.com/covid-projections/covid-projections/pull/3670/files#diff-a4d6872084be7ff9ad51d7175624350349e49a07fae789007996dcb43a793df8R70) are required by the close button (`FilledButton`)